packages/easy-coding-standard/build/target-repository/.gitignore

```diff
@@ -0,0 +1 @@
+composer.lock
```

Comment:
What is this for? It's not needed, as we don't touch composer of split package. It would break the autoload

Reply:
okay, I'll remove this tomorrow